src/main/xsl/common/dita-utilities.xsl

```diff
@@ -12,9 +12,9 @@
 
   <xsl:include href="functions.xsl"/>
 
-  <xsl:param name="ditaot.default.language" select="'en'"/>
+  <xsl:param name="defaultLanguage" select="'en'"/>
 
-  <xsl:param name="DEFAULTLANG" select="if (/*/@xml:lang) then /*/@xml:lang else $ditaot.default.language"/>
+  <xsl:param name="DEFAULTLANG" select="if (/*/@xml:lang) then /*/@xml:lang else $defaultLanguage"/>
```

Comment:
I don't think so. The problem is that `defaultLanguage` should always be passed in from the build, from the value in `configuration.properties`. It represents the default setting for the toolkit. The only place it is / should be used in the code is as a fallback for `$DEFAULTLANG` when no value is found on the map. If we condense to one parameter, the value passed from the scripts will always override the value on the map.